src/FcsWatch.Binary/AutoReload.fs

```diff
@@ -203,9 +203,17 @@ module AutoReload =
                                 projLevelMap.[refCrackedFsproj.ProjPath]
                             )
                             |> Seq.iter (CrackedFsproj.copyFileFromRefDllToBin projPath)
+
+
                         )
 
-                        CrackedFsproj.tryReRun args cache.EntryCrackedFsproj
+                        let updatedDlls =
+                            commonState.CompilerTmp |> Seq.collect (fun projPath ->
+                                let currentCrackedFsproj = cache.ProjectMap.[projPath]
+                                currentCrackedFsproj.AsList |> List.map (fun singleTarget -> singleTarget.TargetPath))
+                            |> Seq.toList
+
+                        CrackedFsproj.tryReRun args cache.EntryCrackedFsproj updatedDlls
```

Comment:
If my understanding is correct this should be the same list as the dlls updated.

My understanding is also that this list of dlls will be _every_ dll in the project, not just those that were updated.

Reply:
Not .dll
A file change means sourcefile change which end with .fs

Reply:
I've updated it to pull from the `CompilerResult` list instead